test/Compiler.Dynamic.Tests/DynamicCompilationTests.cs

```diff
@@ -34,6 +34,7 @@ public static void Initialize(TestContext context)
     [DataRow("test06", "route_url_expressionbuilder.aspx")]
     [DataRow("test07", "redirect_page.aspx")]
     [DataRow("test08", "scripts.aspx")]
+    [DataRow("testForUserControl", "basic_page_with_header.aspx")]
```

Comment:
can you change the test folder to `test09`? I've been just numbering them, with the name of the page being something descriptive about what it is testing, i.e. `testForUserControl/basic_page_with_header.aspx` -> `test09/custom_usercontrol.aspx`

Reply:
I have renamed to basic_page_with_usercontrol.aspx... custom_usercontrol is another beast which we need to experiment too :)